src/components/TabLinks.tsx

```diff
@@ -1,8 +1,12 @@
+import { UrlObject } from "url";
+import { Route } from "next";
+import Link from "next/link";
+
 import cc from "classcat";
 
 export type Tab = {
   label: string;
-  href: string;
+  href: UrlObject | Route<string>;
```

Comment:
TSの恩恵が受けられなくなるので微妙極まりないですが、このコメントと下のコメント合わせて対応すると、httpが無くても大丈夫そうです。

```suggestion
  href: string;
```